<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailedSonPageInStackwidget</class>
 <widget class="QWidget" name="DetailedSonPageInStackwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="backgroundwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#backgroundwidget
{
	background-color: rgb(255, 255, 255);
}

</string>
   </property>
   <widget class="QWidget" name="titlewidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#titlewidget
{
	border-image: url(:/login/image/DetialTitle.jpg);
}
#pushButtonOnTitlewidget
{
	color:rgb(255,255,255);
	border-radius: 20px;
	background-color: rgba(0, 0, 0, 0);
}</string>
    </property>
    <widget class="QPushButton" name="pushButtonOnTitlewidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="InAndOutShowwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#InAndOutShowwidget
{
	background-color: rgb(249, 219, 97);
	border-top-color: rgb(249, 219, 97);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="toolwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.994792, y1:0.313, x2:1, y2:1, stop:0 rgba(249, 219, 97, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_Bill">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/DetialToolButton/image/BillToolButton.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Budget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/DetialToolButton/image/BudGetToolButton.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_AssetManager">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/DetialToolButton/image/AssetManagerToolButton.jpg);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_Return">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/DetialToolButton/image/ShoppingReturnToolButton.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_More">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/DetialToolButton/image/MoreToolButton.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
